wordpress-mobile/WordPress-iOS
# Dark mode: loading animations and no results view text

Refs #12320

This PR updates the colors for the loading animation, and text colors in the no results controller. It also moves `NoResultsViewController.swift` to be in the same folder as its storyboard 😎

![no results dark mode](https://user-images.githubusercontent.com/517257/63457434-e6f1d600-c405-11e9-8c35-68d809c8825a.gif)
<img width="553" alt="image" src="https://user-images.githubusercontent.com/517257/63457472-fa04a600-c405-11e9-91bd-6acdb756ebec.png">


To test:
- View a screen that has to do some loading, like stats.
- Ensure the colors look better
- Test in dark mode, too 🌑

Update release notes:
